docs/sources/visualizations/geomap.md

```diff
@@ -180,4 +181,4 @@ default_baselayer_config = `{
 }`
 ```
 
-`enable_custom_baselayers` allows you to enable or disable custom open source base maps that are already implemented. The default is `true`.
+`enable_custom_baselayers` allows you too enable or disable custom open source base maps that are already implemented. The default is `true`.
```

Comment:
This looks incorrect. Should be `to` and not `too` as it allows you **to** enable or disable.

Reply:
😅 I confuse myself sometimes! Thanks!